### PR TITLE
Cleanup docs about runners

### PR DESCRIPTION
**Pull request by evanphx**
* Many various clarifications
* Clarify `remote runner` => `static runner`
* Remove many mentions of CLI runners being the most common setup

### CODE REVIEW

File: website/content/docs/internals/architecture.mdx
Comment (koesbong) [inline website/content/docs/internals/architecture.mdx:43]:
```suggestion
  * **Static Runner** - A static runner is a long-lived runner that is set up
```

File: website/content/docs/internals/architecture.mdx
Comment (koesbong) [inline website/content/docs/internals/architecture.mdx:44]:
```suggestion
when a team begins using waypoint.
```

File: website/content/docs/internals/architecture.mdx
Comment (koesbong) [inline website/content/docs/internals/architecture.mdx:87]:
```suggestion
`-local=true`, then the CLI will spawn a runner itself.
```

File: website/content/docs/internals/execution.mdx
Comment (koesbong) [inline website/content/docs/internals/execution.mdx:27]:
```suggestion
is executed remotely, which causes the static runner to spawn an on-demand
```

File: website/content/docs/internals/execution.mdx
Comment (koesbong) [inline website/content/docs/internals/execution.mdx:29]:
```suggestion
use Waypoint.
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:20]:
```suggestion
When creating a new static runner, it is possible to generate the runner token and install
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:21]:
```suggestion
a static runner with the token on its initial installation. These runners are then
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:27]:
```suggestion
static runner is rejected, a new runner token must be created for the runner before it
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:26]:
```suggestion
issuing the [`waypoint runner reject`](/commands/runner-reject) command. If a preadopted
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:49]:
```suggestion
adopted, it may start receiving job assignments from the Waypoint server as usual.
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:54]:
```suggestion
[`waypoint runner reject`](/commands/runner-reject) command, the rejected runner 
```

File: website/content/docs/runner/adoption.mdx
Comment (koesbong) [inline website/content/docs/runner/adoption.mdx:77]:
```suggestion
new token from the Waypoint server. First, the runner will look in its state
```

File: website/content/docs/runner/config.mdx
Comment (koesbong) [inline website/content/docs/runner/config.mdx:27]:
```suggestion
a CLI runner when an operation runs locally). To unset any variables, assign it to
```

File: website/content/docs/runner/config.mdx
Comment (koesbong) [inline website/content/docs/runner/config.mdx:35]:
```suggestion
~> **Security note:** HCP Waypoint stores these values encrypted within its backend
```

File: website/content/docs/runner/config.mdx
Comment (koesbong) [inline website/content/docs/runner/config.mdx:46]:
```suggestion
available within certain scoped situations. Unscoped variables are globally sent to
```

File: website/content/docs/runner/index.mdx
Comment (koesbong) [inline website/content/docs/runner/index.mdx:33]:
```suggestion
many operations at a time. Additionally to run certain plugins like Docker, we need to have
```

File: website/content/docs/runner/on-demand-runner.mdx
Comment (koesbong) [inline website/content/docs/runner/on-demand-runner.mdx:20]: should this say `on hold`?

File: website/content/docs/runner/on-demand-runner.mdx
Comment (koesbong) [inline website/content/docs/runner/on-demand-runner.mdx:60]:
```suggestion
The on-demand configuration also accepts a specific OCI image URL to use instead. 
```

File: website/content/docs/runner/profiles.mdx
Comment (koesbong) [inline website/content/docs/runner/profiles.mdx:86]:
```suggestion
It may be necessary to change the runner profile OCI URL. This can be used to upgrade
```

File: website/content/docs/runner/profiles.mdx
Comment (koesbong) [inline website/content/docs/runner/profiles.mdx:90]:
```suggestion
Below is an example of switching the ODR image to a pre-release alpha version of
```

File: website/content/docs/runner/index.mdx
Comment (briancain) [reply website/content/docs/runner/index.mdx:33]: Maybe instead of `we` I'd say `you would`

**Review by briancain — APPROVED** [review]
I agree with all of @koesbong review comments, so I'll approve assuming those get committed! 🖖🏻

File: website/content/docs/runner/on-demand-runner.mdx
Comment (evanphx) [reply website/content/docs/runner/on-demand-runner.mdx:20]: I think `held` is best here.

**Review by briancain — CHANGES_REQUESTED** [review]

**Review by briancain — APPROVED** [review]
Looks good, but I think we have to fix that Vercel error before merge? Looks like a legit one

**Comment by briancain** [comment]
From vercel:

```


Failed to generate static props: YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 6, column 1:
--
15:43:58.918 |  
15:43:58.918 | ^
15:43:58.919 | at generateError (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:167:10)
15:43:58.919 | at throwError (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:173:9)
15:43:58.919 | at readBlockMapping (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:1073:9)
15:43:58.920 | at composeNode (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
15:43:58.920 | at readDocument (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:1525:3)
15:43:58.920 | at loadDocuments (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:1588:5)
15:43:58.921 | at load (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:1614:19)
15:43:58.921 | at Object.safeLoad (/vercel/path1/website-preview/node_modules/js-yaml/lib/js-yaml/loader.js:1637:10)
15:43:58.921 | at module.exports (/vercel/path1/website-preview/node_modules/gray-matter/lib/parse.js:12:17)
15:43:58.921 | at parseMatter (/vercel/path1/website-preview/node_modules/gray-matter/index.js:109:17) {
15:43:58.922 | reason: 'can not read a block mapping entry; a multiline key may not be an implicit key',
15:43:58.922 | mark: Mark {
15:43:58.922 | name: null,
15:43:58.922 | buffer: '\n' +
15:43:58.923 | 'layout: docs\n' +
15:43:58.923 | 'page_title: Runners\n' +
15:43:58.923 | 'description: \|-\n' +
15:43:58.924 | 'Waypoint utilizes "runners" to execute Waypoint operations remotely. Runners are responsible for performing builds and deploys, polling projects for changes, and more.\n' +
15:43:58.924 | '\x00',
15:43:58.924 | position: 218,
15:43:58.924 | line: 5,
15:43:58.924 | column: 0
15:43:58.925 | }
15:43:58.925 | }
15:43:58.925 |  
15:43:58.925 | Error occurred prerendering page "/_proxied-dot-io/waypoint/docs/runner". Read more: https://nextjs.org/docs/messages/prerender-error
```

**Review by koesbong — APPROVED** [review]

File: website/content/docs/runner/index.mdx
Comment (evanphx) [reply website/content/docs/runner/index.mdx:5]: Yeah, I'm fixing that.

**Pull request merged**
